python/pyspark/ml/classification.py

```diff
@@ -273,6 +273,10 @@ def evaluate(self, dataset):
         java_blr_summary = self._call_java("evaluate", dataset)
         return BinaryLogisticRegressionSummary(java_blr_summary)
 
+    @since("2.0.0")
+    def __repr__(self):
```

Comment:
It might make sense to include this in the docstring since it might be useful for people to see how to use it :)